src/nr_openai_observability/build_events.py

```diff
@@ -8,25 +8,42 @@
 
 logger = logging.getLogger("nr_openai_observability")
 
+from nr_openai_observability.call_vars import (
+    get_conversation_id,
+    set_response_model,
+    get_response_model
+)
+
+def build_messages_events(messages, model, completion_id, message_id_override=None, response_id=None, tags={}, start_seq_num=0):
+    if model is not None:
+        set_response_model(model)
 
-def build_messages_events(messages, model, completion_id, tags={}, start_seq_num=0):
     events = []
     for index, message in enumerate(messages):
+        #Bedrock and non-final messages
```

Comment:
Is this case used? Or perhaps the comment is a bit inaccurate? I don't believe the bedrock code uses this method and I did not notice it being added as part of the PR.

Reply:
The bedrock portion of this comment should definitely be removed - the user/system portions of calls for openAI don't have a response_id to generate a `{reponse_id}-{index}` message_id - only the final response returned by the assistant does so this is still used for the first two messages recorded by openAI